# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/TabChild.h
# Commit: 277768e37335
# Full Hash: 277768e37335cef6490ca9d35122af8b231398cd
# Author: Barret Rennie <barret@brennie.ca>
# Date: 2019-04-04 04:33:36
# Regressor Bug: 1510569
# File Overlap Count: 1
# Description:
#   Bug 1510569 - Use Maybe<T> directly in TabChild::PrepareProgressListenerData r=nika,Alex_Gaynor
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D24806
# ==============================================================================

diff -r 1ba9a2cebf12 -r 277768e37335 dom/ipc/TabChild.h
--- a/dom/ipc/TabChild.h	Wed Apr 03 17:29:15 2019 +0000
+++ b/dom/ipc/TabChild.h	Wed Apr 03 17:29:28 2019 +0000
@@ -792,7 +792,7 @@
 
   nsresult PrepareProgressListenerData(nsIWebProgress* aWebProgress,
                                        nsIRequest* aRequest,
-                                       WebProgressData& aWebProgressData,
+                                       Maybe<WebProgressData>& aWebProgressData,
                                        RequestData& aRequestData);
 
   class DelayedDeleteRunnable;